### DISCUSSION TITLE
How to set uniform color theme for representation

### DISCUSSION THREAD

**Discussion by scheuerv** [molstar/molstar#182] [Q&A] [answered]
I'm trying to set uniform color of representation this way:
```
const sel = MS.struct.generator.atomGroups({
    'residue-test': MS.core.rel.inRange([MS.struct.atomProperty.macromolecular.auth_seq_id(), 50, 70])
});
const extraHighlight = await this.plugin.builders.structure.tryCreateComponentFromExpression(structure, sel, 'extra-highlight');
await this.plugin.builders.structure.representation.addRepresentation(extraHighlight!, {
    type: 'molecular-surface', typeParams: { alpha: 0.5 }, color: "uniform", colorParams: { value: PD.Color(Color(0xFF0000)) }
});
```
I would expect red color, but I keep getting this result with black.
![image](https://user-images.githubusercontent.com/61685709/117579634-1a055d00-b0f4-11eb-9f9e-8e3323ba81e3.png)
Is there something I'm doing wrong or is this expected behaviour?
Thanks for any advice.

**Comment by dsehnal** [comment, marked as answer]
Replace `` { value: PD.Color(Color(0xFF0000)) }`` ==> ``{ value: Color(0xFF0000) }`` and it should work.

The ``PD`` (``ParamDefinition``) is metadata for the UI of the property, not a value wrapper.

**Reply by scheuerv** [reply]
Thank you for your help. It works now.
